### ISSUE TITLE
Data ideas

### ISSUE THREAD

**Issue by sharlagelfand** [sharlagelfand/kableExtra-cookbook#7]
We can't make tables without data! A place to post data sets to use within the book.

Some initial sources:
* [Tidy Tuesday](https://github.com/rfordatascience/tidytuesday)
* [The Pudding?](https://pudding.cool/)
* [Animal Crossing :~)](https://github.com/chendaniely/animalcrossing)

**Comment by geanders** [comment]
Datasets that are already in a package might work well, too. Here are a couple of packages with some interesting data: 

- `dslabs`
- `dplyr` (a few, but really interesting)
- `ggplot2` (again, a few, but really interesting)

**Comment by sharlagelfand** [comment]
Yeah data in packages is a great idea! Allows things to be as self contained as possible.

There's also the fivethirtyeight package which has data from FiveThirtyEight stories! those could be a good source of tables too - I'll dig around

https://github.com/rudeboybert/fivethirtyeight

**Comment by geanders** [comment]
Oh, yes, the fivethirtyeight one could be great!
